<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MSRC config</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>70</height>
           </size>
          </property>
          <property name="title">
           <string>Receiver</string>
          </property>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>331</width>
             <height>31</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Protocol</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cbReceiver"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>450</height>
           </size>
          </property>
          <property name="title">
           <string>Sensors</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="gridLayoutWidget_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>331</width>
             <height>411</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="4" column="0">
             <widget class="QCheckBox" name="cbTemperature1">
              <property name="text">
               <string>Temperature 1</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QGroupBox" name="gbAltitude">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>70</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string>Altitude</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <widget class="QWidget" name="horizontalLayoutWidget">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>311</width>
                 <height>31</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>   Address</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cbAddress"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="gbGps">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>70</height>
               </size>
              </property>
              <property name="title">
               <string>GPS</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <widget class="QWidget" name="gridLayoutWidget_4">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>311</width>
                 <height>31</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>   Baud rate</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="cbGpsBaudrate"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="cbVoltage1">
              <property name="text">
               <string>Voltage 1</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QCheckBox" name="cbCurrent">
              <property name="text">
               <string>Current</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QCheckBox" name="cbTemperature2">
              <property name="text">
               <string>Temperature 2</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="cbVoltage2">
              <property name="text">
               <string>Voltage 2</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QCheckBox" name="cbAirspeed">
              <property name="text">
               <string>Airspeed</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="gbEsc">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>70</height>
               </size>
              </property>
              <property name="title">
               <string>ESC</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <widget class="QWidget" name="gridLayoutWidget_2">
               <property name="geometry">
                <rect>
                 <x>10</x>
                 <y>30</y>
                 <width>311</width>
                 <height>31</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_20">
                  <property name="text">
                   <string>   Protocol</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="cbEsc"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btGenerate">
          <property name="text">
           <string>Generate config</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Copy this text to config.h</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="txConfig">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMSRC">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionGenerate_config"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMSRC"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGenerate_config">
   <property name="text">
    <string>Generate config</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
